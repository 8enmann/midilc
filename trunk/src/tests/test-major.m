/* Test major
   Tests creating a major chord from its root.  Library function. 
   @author Fred
   */

main(){
  Sequence s;
  Chord c;
  s = new_sequence();
  
  c = major(C);
  s = s + c;
  set_tempo(30);
  play(s);
}

major(root)
{
	Chord c;
	
	c = new_chord(root);
	
	c = c + (root .+ 4);
	c = c + (root .+ 7);
	return c;
}
