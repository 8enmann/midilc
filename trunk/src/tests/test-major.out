Tempo,30
0,4,60
0,4,64
0,4,67

